src/main/java/com/itspace/job/dto/RegisterUserDto.java

```diff
@@ -0,0 +1,17 @@
+package com.itspace.job.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class RegisterUserDto {
+
+    private long id;
+    private String name;
+    private String email;
+}
```

Comment:
please check the fields of this class
I see there are many fields in the entity class but you just put three of them in this response dto.
please add all required fields which front needs for showing user's data